sdk/digitaltwins/azure-digitaltwins-core/azure/digitaltwins/core/_digitaltwins_client.py

```diff
@@ -541,13 +542,13 @@ def get_event_route(self, event_route_id, **kwargs):
 
     @distributed_trace
     def list_event_routes(self, **kwargs):
-        # type: (**Any) -> ~azure.core.paging.ItemPaged[~azure.digitaltwins.models.EventRoute]
+        # type: (**Any) -> ~azure.core.paging.ItemPaged[DigitalTwinsEventRoute]
         """Retrieves all event routes.
 
         :keyword int results_per_page: The maximum number of items to retrieve per request.
             The server may choose to return less than the requested max.
-        :return: An iterator instance of list of EventRoute.
-        :rtype: ~azure.core.paging.ItemPaged[~azure.digitaltwins.models.EventRoute]
+        :return: An iterator instance of list of DigitalTwinsEventRoute.
+        :rtype: ~azure.core.paging.ItemPaged[DigitalTwinsEventRoute]
```

Comment:
You will need to wrap the result here too, I think something like this should work:
```
return self._client.event_routes.list(
    event_routes_list_options=event_routes_list_options,
    cls=lambda objs: [DigitalTwinsEventRoute._from_generated(e) for e in objs]),
    **kwargs
)
```

Reply:
Not anymore. I renamed the type with a autorest transformation.